package.json

```diff
@@ -13,9 +13,10 @@
     "vitest": "^0.31.2"
   },
   "scripts": {
-    "build": "yarn workspace @rose-lang/core build",
+    "build": "yarn build-ts && yarn workspace @rose-lang/core build",
+    "build-ts": "rimraf packages/core/src/bindings && cargo test --package=rose && cp -r crates/core/bindings packages/core/src/bindings/ && rimraf crates/core/bindings && yarn prettier --write packages/core/src/bindings/",
```

Comment:
Nice! I think you can omit the `prettier` usage here since we're adding those files to `.prettierignore` anyway. Also, could you modify this so that it puts the generated `*.ts` bindings into `packages/wasm/` and not `packages/core/`? We want to keep `"moduleResolution": "node16"` in `packages/core` even if ts-rs prevents us from doing it in `packages/wasm`.

You could also move the second half of this script (after `cargo test`) into `packages/wasm/package.json` if you want since it's kind of similar to the `wbg` script in that package, but either way is fine.